server/www/twitty.html

```diff
@@ -0,0 +1,153 @@
+              data-size="large"
+              data-count="none"
+              data-hashtags="darkpatterns">
+              Tweet</a>
+            </div>
+            <div style="left:30%; top:60%;style=position: absolute;z-index:1100;">
+                <a href="https://twitter.com/share"
+                  class="twitter-share-button"
+                  data-url="https://dl.acm.org/citation.cfm?id=3174108"
+data-text='
+"Interest in critical scholarship that engages with the complexity of user experience (UX) practice is rapidly expanding, yet the vocabulary for describing and assessing criticality in practice is currently lacking'
+                  data-size="large"
+                  data-count="none"
+                  data-hashtags="darkpatterns">
+                  Tweet</a>
+                </div>
+                <div style="left:20%; top:70%;style=position: absolute;z-index:1100;">
+                    <a href="https://twitter.com/share"
+                      class="twitter-share-button"
+                      data-url="https://link.springer.com/chapter/10.1007/978-3-319-96906-0_17"
+data-text="
+'As our lives have moved more and more into digitally mediated spaces, we are subject to more and more interlocking contracts we sign in order to access technology—the terms of service... Reading an average privacy policy should take 30 min based on average reading speed, but instead took 73 s, where terms of service took 51 s yet should have taken 16 min
+(Obar & Oeldorf-Hirsch, 2016).'"
+                      data-size="large"
+                      data-count="none"
+                      data-hashtags="darkpatterns">
+                      Tweet</a>
+                    </div>
+                    <div style="left:9%; top:80%;style=position: absolute;z-index:1100;">
+                        <a href="https://twitter.com/share"
+                          class="twitter-share-button"
+                          data-url="https://en.wikipedia.org/wiki/Informed_consent"
+                          data-text="(What would ur data doppelganger say?)"
+                          data-size="large"
+                          data-count="none"
+                          data-hashtags="fuckzuck">
+                          Tweet</a>
+                        </div>
+<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';
+    if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';
+    fjs.parentNode.insertBefore
+    (js,fjs);}}(document, 'script', 'twitter-wjs');
+  </script>
+
+
+<input
+  style="left:80%; top:80%; style=position: absolute; z-index:1200;"
+    type="button"
+    id = 'homebutton'
+    onclick="location.href='./obstaclecourse.html';"
+    value="Back to ToolKit" />
+
+
+
+<script src="./js/p5.min.js"></script>
```

Comment:
this is the main issue with why the fountain is no longer working. u are referencing a file called `p5.min.js` inside of the `js` folder, but no such file exists. u might have forgotten to copy+paste it when u moved the other files over